packages/rx_bloc_cli/mason_templates/bricks/feature_deeplink/__brick__/lib/l10n/network_lookup/enter_message_lookup.dart

```diff
@@ -0,0 +1,36 @@
+{{> licence.dart}}
```

Comment:
Do we really need lookup for the enter message? Isn't it part of the deep link feature ?